BUILD FAILED

I got this error

/platforms/android/src/by/chemerisuk/cordova/firebase/FirebaseAnalyticsPlugin.java:89: error: illegal start of expression

        this.firebaseAnalytics.setAnalyticsCollectionEnabled(enabled === "true");
                                                                       ^

1 error

and this one too

/ionic/my132/platforms/android/src/by/chemerisuk/cordova/firebase/FirebaseAnalyticsPlugin.java:98: error: <identifier> expected

    private void setCurrentScreen(callbackContext, String name) {

                                                 ^

1 error

--- a/src/android/FirebaseAnalyticsPlugin.java
+++ b/src/android/FirebaseAnalyticsPlugin.java
@@ -95,7 +95,7 @@ public class FirebaseAnalyticsPlugin extends CordovaPlugin {
         callbackContext.success();
     }
 
-    private void setCurrentScreen(callbackContext, String name) {
+    private void setCurrentScreen(CallbackContext callbackContext, String name) {
         this.firebaseAnalytics.setCurrentScreen(
             this.cordova.getActivity(), name, null);
 

